src/Compilers/Core/Portable/CommandLine/CommandLineSourceFile.cs

```diff
@@ -12,15 +12,13 @@ namespace Microsoft.CodeAnalysis
     [DebuggerDisplay("{Path,nq}")]
     public struct CommandLineSourceFile
     {
-        private readonly string _path;
-        private readonly bool _isScript;
-
-        public CommandLineSourceFile(string path, bool isScript)
```

Comment:
>public CommandLineSourceFile(string path, bool isScript) [](start = 8, length = 56)

We should add a constructor rather than replacing the existing public constructor.